junit5/src/test/java/org/jboss/weld/junit5/auto/ScannedClassesAreNotForcedBeansTest.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.weld.junit5.auto;
+
+
+import org.jboss.weld.junit5.auto.beans.ConstructedV8;
+import org.jboss.weld.junit5.auto.beans.V8;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import javax.enterprise.inject.Produces;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+@EnableAutoWeld
+@AddBeanClasses(ConstructedV8.class)
```

Comment:
Wasn't this supposed to be `@AddBeanClasses(InjectedV8.class)` ?
Since as it stand, `ScannedParameterClassesAreNotForcedBeansTest` does exactly the same thing.

Reply:
Correct. Good catch!